src/components/auth/Admin.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import LoginBox from './LoginBox.js';
+import RegisterBox from './RegisterBox.js';
+import './auth.css';
+
+class Admin extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isLoginOpen: true,
+            isRegisterOpen: false
+        }
+    }
+
+    showLoginBox() {
+        this.setState({ isLoginOpen: true, isRegisterOpen: false });
+    }
+
+    showRegisterBox() {
+        this.setState({ isRegisterOpen: true, isLoginOpen: false });
+    }
+
+    render() {
+
+        return (
+            <div>
+                <div className="box-controller">
+                    <div
+                        className={"controller " + (this.state.isLoginOpen
```

Comment:
I prefer to have any logic outside the rendering function. basically make a variable for the classname and apply to the html. same with the hide and show logic